votebot.go

```diff
@@ -62,7 +63,13 @@ func initBot(ctx context.Context, handler *vocdoniHandler, api farcasterapi.API,
 					"electionID", electionID,
 					"poll", poll)
 				frameUrl := fmt.Sprintf("%s/%s", serverURL, electionID.String())
-				if err := voteBot.ReplyWithPollURL(ctx, msg, frameUrl); err != nil {
+				shortenedUrl, err := shortener.ShortURL(ctx, frameUrl)
+				if err != nil {
+					log.Errorf("error shortening election url: %s", err)
```

Comment:
If there is an error with the shortener service, since it is external, I think we should fallback to the full URL.